app-admin/cdist/cdist-6.5.4.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7} )
```

Comment:
Any chance to try with python3.8 so this won't have to be updated immediately again?

Reply:
Pushed it, but didn't test.

Reply:
Seems to work with Python 3.8 as well.